test/util/fixtures/IocMapUtility.java

```diff
@@ -0,0 +1,10 @@
+package util.fixtures;
+
+public final class IocMapUtility {
+    public static String key1="TestKey1";
```

Comment:
This will not change ever, so:
```
public static final String TESTING_KEY_1="testing-key-1";
public static final String TESTING_KEY_2="testing-key-2";
.
.
.
```

Reply:
solved